tests/Frontend/e2e/RoomsIndexCreateNewRoom.cy.js

```diff
@@ -392,6 +404,46 @@ describe("Rooms index create new room", function () {
       .should("be.visible")
       .and("include.text", "The Room type field is required.");
 
+    // Create new room with invalid room type
+    cy.intercept("POST", "api/v1/rooms", {
+      statusCode: 422,
+      body: {
+        message: "The selected Room type is invalid.",
+        errors: {
+          room_type: ["The selected Room type is invalid."],
+        },
+      },
+    }).as("createRoomRequest");
+
+    cy.intercept("GET", "api/v1/roomTypes*", { fixture: "roomTypes.json" }).as(
+      "roomTypeRequest",
+    );
+
+    cy.get('[data-test="room-create-dialog"]')
+      .should("be.visible")
+      .within(() => {
+        // Select a room type
+        cy.get('[data-test="room-type-select-option"]').eq(0).click();
+        // Check that room type is selected
+        cy.get('[data-test="room-type-details"] > h3').contains("Invalid Type");
+
+        // Create new room
+        cy.get('[data-test="dialog-save-button"]').click();
+      });
+
+    cy.wait("@createRoomRequest");
+    cy.wait("@roomTypeRequest");
+
+    // Check that error gets displayed
+    cy.get('[data-test="room-create-dialog"]')
+      .should("be.visible")
+      .and("include.text", "The selected Room type is invalid.");
+
+    // Check refreshed listed is shown and room type select is reset
```

Comment:
```suggestion
    // Check refreshed list is shown and room type select is reset
```

Maybe change this comment